.github/workflows/pull-request-checks.yaml

```diff
@@ -60,6 +60,11 @@ jobs:
           cmake .. -G Ninja -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER=/usr/bin/gcc -DCMAKE_CXX_COMPILER=/usr/bin/g++
       - name: Build with Ninja
         run: cd build; ninja
+      - name: Check if package building works
+        run: |
+          cd build
+          ninja package
+          ls *.deb
```

Comment:
What does the `ls *.deb` achieve, other than a nice output? Will it fail the build in someway? Or will it mask any failures from the `ninja package` because the exit code of the `ls` will be success?

Reply:
My experience of debugging the run sections of github actions suggests that it will stop on the first line where an error code is returned. Similar to writing a bash script where `set -e` has been specified.